### PR TITLE
feat: add responsive image according to hovered UIButton

### PR DESCRIPTION
**Pull request by stefaniamak**
### Tasks
- [x] set desired max and min size for the column layout holding the UIImages

### Purpose

Set an arrow asset to appear in the appropriate UIImage next to the hoved UIButton.

### How to test

- Check https://github.com/Terasology/LightAndShadow/pull/139
- Check https://github.com/Terasology/Dialogs/pull/15
- Start a LAS game
- Talk to The Fool NPC
- See the image/s next to the answer button/s

_note: the extra response and large text at the dialogue box is not committed and is not planned to be_
image

### CODE REVIEW

File: assets/skins/dialogScreen.skin [hunk 1]
```diff
@@ -36,6 +36,9 @@
         },
         "ScrollableArea": {
             "background": ""
+        },
+        "UIImage": {
+            "background": "heartsIcon"
```
Comment (skaldarnar) [inline assets/skins/dialogScreen.skin]: I don't think we should work with a background image for the `UIImage` widget here. 

Note that the background image is rendered behind the widget itself (i.e., the background). The `UIImage` widget would render another image on top if properly configured. I think we should rather set the source for the image to render in the widget correctly.

**Review by skaldarnar — CHANGES_REQUESTED** [review]
After having a closer look at this intended feature I think there are **no** changes needed in LaSR.

File: assets/skins/dialogScreen.skin (hunk 1 above)
Comment (stefaniamak) [reply assets/skins/dialogScreen.skin]: That sounds logical, yes, having a background image should be the way we go about. I'll proceed on removing this line when I work on this task 👍 (or maybe closing that PR will work as well 🤔)

**Review by Cervator — APPROVED** [review]
Have tested this out along with the two other related PRs - seems fine! 👍 

Clearly a fair bit of back-and-forth review cycles have happened with a bunch of commits with relatively few changes, so I'm going to opt for squash merging these. Changes seem relatively straight forward and nothing really jumps out at me :-)

Nicely done!

**Pull request merged**